client/src/components/RequestAccess/RequestAccess.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { Navbar } from 'reactstrap';
+import RequestAccessForm from './RequestAccessForm';
+import BrowserChecker from '../_common/BrowserChecker';
+import './RequestAccess.module.scss';
+
+function RequestAccess() {
+  return (
+    <>
+      <Navbar styleName="request-access-title">Request Access</Navbar>
+      <div styleName="request-access-wrapper">
+        <h2 styleName="request-access-h2">
+          Request Access to the 3Dem Portal Platform
```

Comment:
Anywhere `3Dem` is mentioned, let's instead use the `portalName` attribute of the `workbench` state to make this page applicable to any portal